integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -565,7 +563,9 @@ class CarbonFileMetastore extends CarbonMetaStore {
             tableModifiedTimeStore.get(CarbonCommonConstants.DATABASE_DEFAULT_NAME))) {
         metadata.carbonTables = metadata.carbonTables.filterNot(
           table => table.getTableName.equalsIgnoreCase(tableIdentifier.table) &&
-        table.getDatabaseName.equalsIgnoreCase(tableIdentifier.database.getOrElse("default")))
+                   table.getDatabaseName.equalsIgnoreCase(
```

Comment:
Why it coming here for unmanged table

Reply:
Ran all the test case. It is not coming now. Removed this changes